.cicd/pipeline.yml

```diff
@@ -154,6 +154,75 @@ steps:
 
   - wait
 
+  - label: ":aws: Amazon_Linux 2 - Toolchain Tests"
```

Comment:
If the new toolchain tests are not dependent on the unit tests and you wanted to save some time in CI, you could reorganize `.cicd/pipeline.yml` to run all of the tests together before running the test metrics step. Everything looks good to me though!